src/node_crypto.cc

```diff
@@ -5193,12 +5194,18 @@ void Initialize(Local<Object> target,
 #endif
 
   env->SetMethod(target, "pbkdf2", PBKDF2);
-  env->SetMethod(target, "randomBytes", RandomBytes);
-  env->SetMethod(target, "timingSafeEqual", TimingSafeEqual);
-  env->SetMethod(target, "getSSLCiphers", GetSSLCiphers);
-  env->SetMethod(target, "getCiphers", GetCiphers);
-  env->SetMethod(target, "getHashes", GetHashes);
-  env->SetMethod(target, "getCurves", GetCurves);
+  env->SetMethod(target, "randomBytes", RandomBytes,
+                 SideEffectType::kHasNoSideEffect);
```

Comment:
Is there a reason you annotated randomBytes but not pbkdf2 and scrypt?  They all use the thread pool.

Reply:
Hmm, I didn't realize `RandomBytes` was async. I'll remove this part then.